pkg/aws/awsmachinedeployment/validate_awsmachinedeployment.go

```diff
@@ -125,12 +125,12 @@ func (v *Validator) MachineDeploymentLabelMatch(awsMachineDeployment infrastruct
 
 func (v *Validator) MachineDeploymentAnnotationMaxBatchSizeIsValid(awsMachineDeployment infrastructurev1alpha2.AWSMachineDeployment) error {
 	if maxBatchSize, ok := awsMachineDeployment.GetAnnotations()[aws.AnnotationUpdateMaxBatchSize]; ok {
-		if !aws.MaxBatchSizeIsValid(maxBatchSize, awsMachineDeployment.Spec.NodePool.Scaling.Max) {
-			v.logger.Log("level", "debug", "message", fmt.Sprintf("AWSMachineDeployment annotation '%s' value '%s' is not valid. Allowed value is either positive integer number smaller than number of nodes or decimal number between 0 and 1.0 defining percentage of nodes",
+		if !aws.MaxBatchSizeIsValid(maxBatchSize) {
```

Comment:
I figured out that I does not make sense to limit the maximum number for maxBatchSize as if you set it on AWSCluster you have no idea how each nodePool is big and AWS accept values that can be over the max number of nodes, so removing this constriction from both AWSCluster and AWSMachine deployment